be/src/storage/rowset_update_state.cpp

```diff
@@ -232,20 +232,110 @@ Status RowsetUpdateState::_prepare_partial_update_states(Tablet* tablet, Rowset*
         size_t num_default = 0;
         std::map<uint32_t, std::vector<uint32_t>> rowids_by_rssid;
         vector<uint32_t> idxes;
-        plan_read_by_rssid(_parital_update_states[i].src_rss_rowids, &num_default, &rowids_by_rssid, &idxes);
+        plan_read_by_rssid(_partial_update_states[i].src_rss_rowids, &num_default, &rowids_by_rssid, &idxes);
         // get column values by rowid, also get default values if needed
         RETURN_IF_ERROR(
                 tablet->updates()->get_column_values(read_column_ids, num_default > 0, rowids_by_rssid, &read_columns));
         for (size_t col_idx = 0; col_idx < read_column_ids.size(); col_idx++) {
-            _parital_update_states[i].write_columns[col_idx]->append_selective(*read_columns[col_idx], idxes.data(), 0,
+            _partial_update_states[i].write_columns[col_idx]->append_selective(*read_columns[col_idx], idxes.data(), 0,
                                                                                idxes.size());
         }
     }
 
     return Status::OK();
 }
 
-Status RowsetUpdateState::apply(Tablet* tablet, Rowset* rowset, uint32_t rowset_id, const PrimaryIndex& index) {
+Status RowsetUpdateState::_check_conflict(Tablet* tablet, Rowset* rowset, uint32_t rowset_id,
```

Comment:
this method not only check conflict, but also resolve the conflict, so maybe change to _check_and_resolve_conflict ?